packages/firestore-types/index.d.ts

```diff
@@ -468,13 +468,14 @@ export class WriteBatch {
 }
 
 /**
- * Options for use with `DocumentReference.onSnapshot()` to control the
- * behavior of the snapshot listener.
+ * An options object that can be passed to `DocumentReference.onSnapshot()`,
+ * `Query.onSnapshot()` and `QuerySnapshot.docChanges()` to control which
+ * types of changes to include in the result set.
  */
-export interface DocumentListenOptions {
+export interface SnapshotListenOptions {
   /**
-   * Raise an event even if only metadata of the document changed. Default is
-   * false.
+   * Include a change even if only the metadata of the query or a document
```

Comment:
I would like to make this:

`Include a change even if only the metadata of the query or OF a document changed.`

Let me know if you can take care of this or if I should create a new PR against the branch.